### ISSUE TITLE
False positive for `MultilineMethodCallIndentation` with `[]` on the left side of assignment

### ISSUE THREAD

**Issue by AlexWayfer** [rubocop-hq/rubocop#7338]
There is false positive for `MultilineMethodCallIndentation` with `[]` on the left side of assignment.

--------

## Expected behavior

Without offenses.

## Actual behavior

With offenses.

## Steps to reproduce the problem

```yaml
# .rubocop.yml
Layout/MultilineMethodCallIndentation:
  EnforcedStyle: indented
```

```ruby
# test.rb
# frozen_string_literal: true

params[:key] =
  foo
    .bar
    .baz
```

```
> bundle exec rubocop test.rb
Inspecting 1 file
C

Offenses:

test.rb:5:5: C: Layout/MultilineMethodCallIndentation: Use 2 (not 4) spaces for indenting an expression in an assignment spanning multiple lines.
    .bar
    ^^^^
test.rb:6:5: C: Layout/MultilineMethodCallIndentation: Use 2 (not 4) spaces for indenting an expression in an assignment spanning multiple lines.
    .baz
    ^^^^
```

## RuboCop version

Include the output of `rubocop -V` or `bundle exec rubocop -V` if using Bundler. Here's an example:

```
0.74.0 (using Parser 2.6.4.0, running on ruby 2.6.4 x86_64-linux)
```

**Comment by AlexWayfer** [comment]
There is probably related #7726.

**Comment by AlexWayfer** [comment]
OK, another one unresolved closed issue. 🙇‍♂️
